pkg/controller/controller_broker_test.go

```diff
@@ -185,6 +188,9 @@ func TestReconcileBrokerExistingServiceClass(t *testing.T) {
 }
 
 func TestReconcileBrokerExistingServiceClassDifferentExternalID(t *testing.T) {
```

Comment:
as function godoc, yes. 
I like the "returns an error " part, as the function . name does not show that.